### DISCUSSION TITLE
Why isn't this included as a default install?

### DISCUSSION THREAD

**Discussion by TheJaysH** [microsoft/winget-cli#2241] [Q&A]
Genuinely curious why this isn't included in the base windows image. This would make provisioning so much easier. Its a shame its not supported on 2019 however its understandable if API's are not present. 

At the very least having it as a windows feature that could be enabled would be great.

**Comment by Masamune3210** [comment]
There is a good chance it will be at some point part of the base image, it just hasn't caught up to the latest version of the App Installer package which is what brings in winget
